framework/CodeInterfaces/PHISICS/phisicsdata.py

```diff
@@ -0,0 +1,961 @@
+        self.writeCSV(instantDict,timeStepIndex,mrtauTimeSteps,phisicsDataDict['jobTitle'])
+
+      if phisicsDataDict['mrtauStandAlone'] == True:
+        decayHeatMrtau = self.getDecayHeatMrtau(timeStepIndex, mrtauTimeSteps)
+        depList = self.getDepInfoMrtau(timeStepIndex, mrtauTimeSteps)
+        mrtauDict['workingDir'] = self.workingDir
+        mrtauDict['decayHeatMrtau'] = decayHeatMrtau
+        mrtauDict['depList'] = depList
+        mrtauDict['timeStepIndex'] = timeStepIndex
+        mrtauDict['mrtauTimeSteps'] = mrtauTimeSteps
+        self.writeMrtauCSV(mrtauDict)
+  
+  def summedListValues(self,list):
+    """
+      Sums up values from a given list.
+      @ In, list, list of intergers or floats
+      @ Out, sum, integer or float, sum of the values given in input
+    """
+    sum = 0
+    for i in xrange (0,len(list)):
+      sum = sum + float(list[i])
+    return sum
+  
+  def summedDictValues(self,dict,typeOfParameter,parameterList):
+    """
+      Sums up the values from a given dictionary.
+      @ In, dict, dictionary, nested dictionaries of intergers or floats
+      @ In, typeOfParameter, string, either 'reactionRates' or 'FissionMatrix'
+      @ In, parameterList, list, list of the parameters parsed
+      @ Out, summedDict, dict, dictionary of integer or float
+    """
+    summedDict = self.declareDict(typeOfParameter,parameterList)
+    if self.printSpatialRR is False:
+      for parameter,mpi in dict.iteritems():
+        sum = 0
+        for mpi,value in mpi.iteritems():
+          sum = '%.15E' % Decimal(float(sum) + float(value))
+        summedDict[parameter] = sum
+    if self.printSpatialRR is True:
+      for group,region in dict.iteritems():
+        for region,parameter in region.iteritems():
+          for parameter,mpi in parameter.iteritems():
+            sum = 0
+            for mpi,value in mpi.iteritems():
+              sum = '%.15E' % Decimal(float(sum) + float(value))
+            summedDict[group][region][parameter] = sum
+    return summedDict
+      
+  def fileOutName(self,phisicsDataDict):
+    """
+      Puts in a list the INSTANT output file names based on the number of MPI.
+      The format is title.o-MPI
+      @ In, phisicsDataDict, dictionary, dictionary of variables passed by the interface
+      @ Out, instantOutputFileMPI, list, list of INSTANT output file names
+      @ Out, mrtauOutputFileMPI, list, list of MRTAU output file names
+    """
+    instantMPI = []
+    mrtauMPI = []
+    for mpi in xrange (0,phisicsDataDict['numberOfMPI']):
+      mrtauMPI.append(os.path.join(self.workingDir,phisicsDataDict['mrtauFileNameDict']['atoms_csv'].split('.')[0]+'-'+str(mpi)+'.csv')) # MRTAU files (numbers.csv)
+      if self.phisicsRelap is False:    # INSTANT files, no coupling with RELAP
+          instantMPI.append(phisicsDataDict['instantOutput']+'-'+str(mpi))
+      if self.phisicsRelap is True:     # INSTANT files, coupling with RELAP
+        if mpi == 0:
+          pass
```

Comment:
pass? 

```
if mpi>0:
      instantMPI.append('INSTout-'+str(mpi))
```

Reply:
Yes it is a way to ignore the first increment in PHISICS?RELAP coupling mode. 
The naming is different if you use PHISICS as compared to PHISICS/RELAP (name-0.o does not exist in PHISICS/RELAP), hence the exception.